configs/benchmarks/classification/_base_/models/vit_base_16_linear_eval.py

```diff
@@ -0,0 +1,10 @@
+model = dict(
+    type='MAELinearClassification',
+    backbone=dict(
+        type='MAEClsViT',
+        arch='b',
+        patch_size=16,
+        global_pool=True,
+        finetune=False,
+        final_norm=False),
+    head=dict(type='MAELinearEvalHead', num_classes=1000, embed_dim=768))
```

Comment:
Name consistency? `linear_eval`, `MAELinearClassification` and `MAELinearEvalHead` are not consistent, can we name them with the same convention?
Besides, I think the `vit-base-p16_linear-eval.py` format is better.

Reply:
Thank you for your valuable feedbacks. Your suggestion is reasonable and I will fix it.